cli/src/installers/envVars.ts

```diff
@@ -37,8 +37,9 @@ DATABASE_URL=file:./db.sqlite
     envContent += `
 # Next Auth
 # You can generate the secret via 'openssl rand -base64 32' on Linux
+# Make sure not to keep the secret empty
```

Comment:
don't think we need this if we are including a default

Reply:
removed (https://github.com/t3-oss/create-t3-app/pull/719/commits/0d284110f835d268c45d43fd03e8aa6c291a07b8)